Fix CSS for emoji sprites.

showell opened the pull request:
newer version

showell commented:
About to update some comments and tweak...

showell commented:
Ok, re-pushed.

andersk commented:
Any functional change to `build_emoji` needs a `PROVISION_VERSION` bump.

andersk commented on tools/setup/emoji/build_emoji:
```diff
@@ -110,14 +124,15 @@ def generate_sprite_css_files(cache_path: str,
-                'pos_x': (emoji["sheet_x"] * 100) / max_dim,
-                'pos_y': (emoji["sheet_y"] * 100) / max_dim,
+                'pos_x': percent(emoji["sheet_x"] / max_dim),
+                'pos_y': percent(emoji["sheet_y"] / max_dim),
```
Accounting for gutters, I think this actually needs to be `percent((emoji["sheet_x"] * 66 + 1) / (max_dim * 66 + 2))`.

showell replied:
I re-pushed so that I no longer add that 66/64 factor for background-size, which would have been a functionality change, and which would have required the adjustment you mention.  In doing that I also killed off `max_dim` and added more comments explaining various calculations.

showell commented:
Latest updates:
- added provision bump (which Tim will need to possibly adjust for final merges)
- removed dubious gutter-related tweak from prior push
- added new commit to extract ugly code for square size

timabbott commented:
This is great, merged, thanks @showell!  I did a diff against the previous `static/generated/emoji` content, and there's basically just the reduction in sig figs in most places (and 3 extra 0s for the main background-size).  Seems fine and fixes the bug.  I also appreciate the comments explaining what's going on.
